ui/src/app/components/object-detail/object-detail.component.html

```diff
@@ -662,7 +662,7 @@ <h3 class="title">
             this.interleaveParentName === null
           "
           mat-raised-button
-          color="primary"
```

Comment:
Some thing applies here in this file. Please add back **color="primary"**

Reply:
Made the suggested changes